x-pack/plugins/observability_solution/observability/server/lib/rules/custom_threshold/custom_threshold_executor.test.ts

```diff
@@ -1622,25 +1622,6 @@ describe('The custom threshold alert type', () => {
       });
       await execute(COMPARATORS.GREATER_THAN, [0.9]);
       const ISO_DATE_REGEX = /^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.\d{3}Z$/;
-      expect(services.alertsClient.setAlertData).toBeCalledTimes(1);
-      expect(services.alertsClient.setAlertData).toBeCalledWith({
```

Comment:
Removed as we don't need to provide the ECS groups for recovered alerts as the data is already in the alert document.